### ISSUE TITLE
foundation passport problems with Specter Desktop multisig

### ISSUE THREAD

**Issue by riannom** [Foundation-Devices/passport-firmware#6]
does the passport currently work with multisig on Specter Desktop?  
I was able to add the device under "Other" in specter desktop and create a multisig wallet as well.

I am not able to use the QR code generated by Specter and scan/sign a transaction on the Passport.  The camera/device doesn't seem to read/acknowledge the QR code.  no error or success message.

I am also not able import the psbt transaction via SD card.  When i try to sign it on the passport i get an error "invalid psbt module object has no attribute codecs. multisig.py:1039".

Also is there is a way to view the xpub/ypub/zpub on the device to verify as well?

Lastly, are there plans to get the Foundation Passport into Specter as an actual device option rather than as an "Other" wallet?
Thanks!

**Comment by zachherbert** [comment]
Apologies for the delay @riannom. We are aware of some issues with Specter. Most are resolved if you choose "Cobo" as the device type, but we don't think this is a good user experience. We also had to temporarily disable the ability to pair to Specter via QR codes, as they are unable to parse multi-QR codes (formatted as UR1.0 or UR2.0) to share the XPUBs. 

We are working on getting an official integration and device option in Specter, which will resolve all these issues.

For now, in our v1.0.3 release (coming out today), we are removing the Specter option in our Pair Wallet menu. Hopefully we will be able to bring it back in our next release!

There currently is no quick way to view the ypub or zpub; only the master xpub (in the About menu). This is on our near-term roadmap to add.

**Comment by sako0938** [comment]
I feel like this issue should be re-opened until Specter Desktop multisig is properly working, just so people can follow along with the progress.

Ive been fighting to get it to work today, using firmware version 1.0.4, and creating the device in specter as a Cobo wallet device. I am using 2 coldcards and 1 Passport in my quorum.

In Specter desktop, I am running the animated QR code, and the Passport is able to read it (these animated codes are pretty cool!)

Currently stuck on the error: "xfp depth=1 wrong", I also got a similar issue when doing it with a Microsd card.

**Comment by zachherbert** [comment]
You're absolutely right @sako0938 we will keep it open until there is an official integration. Most things work if you use Passport as a Cobo but there are some edge cases we need to iron out. An official integration will solve this.

**Comment by coreylakey** [comment]
Closing this issue now since Specter/Passport support has been added!

Specter has added Passport support: (https://github.com/cryptoadvance/specter-desktop/commit/94223d4b3c70de1479371566f360fd1da35fcf6b)

Passport has added Specter support: (https://github.com/Foundation-Devices/passport-firmware/pull/44)
